pydantic/main.py

```diff
@@ -920,9 +926,10 @@ def create_model(
                 f_annotation, f_value = f_def
             except ValueError as e:
                 raise PydanticUserError(
-                    'field definitions should either be a tuple of (<type>, <default>) or just a '
+                    'Field definitions should either be a tuple of (<type>, <default>) or just a '
                     'default value, unfortunately this means tuples as '
-                    'default values are not allowed'
+                    'default values are not allowed',
```

Comment:
Random thing that maybe we can fix now — we never actually allow non-tuple field definitions now (because we don't allow non-annotated fields). So this error message can (and probably should) be updated. If you don't want to deal with changing that in this PR maybe we can still just insert a TODO comment into this PR to fix the message?

Reply:
done.